src/amo/utils/errors.js

```diff
@@ -1,16 +1,16 @@
 /* @flow */
-import NotAuthorized from 'amo/components/ErrorPage/NotAuthorized';
-import NotFound from 'amo/components/ErrorPage/NotFound';
-import ServerError from 'amo/components/ErrorPage/ServerError';
+import NotAuthorizedPage from 'amo/pages/ErrorPages/NotAuthorizedPage';
+import NotFoundPage from 'amo/pages/ErrorPages/NotFoundPage';
+import ServerErrorPage from 'amo/pages/ErrorPages/ServerErrorPage';
 
 export function getErrorComponent(status: number | null) {
```

Comment:
It would be dangerous if someone used this thinking that it returned error components rather than *pages*.

I suggest the following:
* Rename it to `getErrorPage`
* Move the function into `amo/components/App` because that's the only place it's ever used. This will also help to discourage usage.